Lib/gftools/builder/__init__.py

```diff
@@ -449,12 +449,12 @@ def instantiate_static_fonts(self, directory, postprocessor):
 
     def build_a_static_format(self, format, directory, postprocessor):
         self.mkdir(directory, clean=True)
-        args = {
-            "output": [format],
-            "output_dir": directory,
-            "optimize_cff": CFFOptimization.SUBROUTINIZE,
-        }
         for source in self.config["sources"]:
+            args = {
+                "output": [format],
+                "output_dir": directory,
+                "optimize_cff": CFFOptimization.SUBROUTINIZE,
+            }
             if source.endswith("ufo"):
                 if "interpolate" in args:
```

Comment:
This can probably go now, since there's no way for `interpolate` to get into `args`.

Reply:
The `interpolate` argument in fontmake is a strange one. If I remove this arg, it won't generate glyphsapp instances for glyphsapp sources.

from --help:
```
 -i [INSTANCE_NAME], --interpolate [INSTANCE_NAME]
                        Interpolate masters and generate all the instances
                        defined. To only interpolate a specific instance (or
                        instances) that match a given "name" attribute, you
                        can pass as argument the full instance name or a
                        regular expression. E.g.: -i "Noto Sans Bold"; or -i
                        ".* UI Condensed". (for Glyphs or MutatorMath sources
                        only).

```

I'm going to change the code from:

```PYTHON
            if source.endswith("ufo"):
                if "interpolate" in args:
                    del args["interpolate"]
            else:
                 args["interpolate"] = True
```

to: 

```PYTHON
           if source.endswith("glyphs"):
               args["interpolate"] = True
```

This way we still get glyphsapp instances